scraper/src/mindtouch2zim/asset.py

```diff
@@ -43,9 +43,15 @@ class HeaderData(NamedTuple):
 
 
 class AssetDetails(NamedTuple):
-    urls: set[HttpUrl]
+    asset_urls: set[HttpUrl]
```

Comment:
This needs a docstring now that it gets more complicated.
Actually, I think this would help decide if this is a good way to implement this. It looks quite weird to me to store here the used_by data which is purely dev/logging and it's poorly connected to the assets url.
I think I'm missing the big picture here